### ISSUE TITLE
Regarding your other project obs-studio-extra-tracks

### ISSUE THREAD

**Issue by successmiven** [aayla-secura/wp-tests#1]
I'm sorry I can't make issue under project [obs-studio-extra-tracks](https://github.com/aayla-secura/obs-studio-extra-tracks). 

  I downloaded and tried your build file, and the UI shows that it supports 12 tracks. The log also shows that 12 tracks are being recorded, and the generated video file also has 12 tracks, but only 8 tracks are available. I have also made modifications in the latest code of OBS before, and only 8 tracks are available. Is it okay for you to use it?

  Wishing you a happy life.

**Comment by aayla-secura** [comment]
Hi there, I've heard of other people experiencing this issue (seeing only 8 tracks having sound and 9 to 12 being silent). Unfortunately I can't reproduce this on my machine. For me all 12 tracks work properly. As I mentioned in the repository, I do not plan on maintaining this, hence issues are closed. Sorry about that.

**Comment by successmiven** [comment]
I change this file.
obs-studio\libobs\obs-source.c

//source->audio_mixers = 0xFF;
source->audio_mixers = 0x1FFFFF;

**Comment by aayla-secura** [comment]
Thank you for the suggested fix. However when I rebuilt it using this value, my audio input was no longer working (no error, just no sound input), whereas previously it was working.

Can I ask, when you initially tried it and saw only 8 channels, did you compile and build it yourself on your machine, or did you use the precompiled zip I had uploaded? If you used my precompiled zip I can only guess that the reason it wasn't working for you, but was for me (I had 12 sound on channels), is that something in my build environment is making it generate code that's specific to my environment and not really portable, if that makes sense.
